crates/wasi/src/preview2/preview1/mod.rs

```diff
@@ -317,23 +317,18 @@ trait WasiPreview1ViewExt:
 
 impl<T: WasiPreview1View + preopens::Host> WasiPreview1ViewExt for T {}
 
-pub fn add_to_linker<
-    T: WasiPreview1View
-        + bindings::cli_base::environment::Host
-        + bindings::cli_base::exit::Host
-        + bindings::filesystem::types::Host
-        + bindings::filesystem::preopens::Host
-        + bindings::sync_io::poll::poll::Host
-        + bindings::random::random::Host
-        + bindings::io::streams::Host
-        + bindings::clocks::monotonic_clock::Host
-        + bindings::clocks::wall_clock::Host,
->(
+pub fn add_to_linker_async<T: WasiPreview1View>(
```

Comment:
Ty, this set of traits got even worse with my cli world update, your change is the right way to handle it.